docs/upload-your-snap.md

```diff
@@ -0,0 +1,55 @@
+# Upload to the store
+
+So you've been working hard on your .snap, and you finally have it to the point
+where you're ready to share it with the world? Great! You can use Snapcraft to
+upload it directly to the store.
+
+
+## Authenticate to the store
+
+First of all, you need to give Snapcraft permission to communicate with the
+store on your behalf:
+
+    $ snapcraft login
+    Enter your Ubuntu One SSO credentials.
+    Email: me@example.com
+    Password:
+    OTP: <press enter here if you haven't enabled two-factor authentication>
```

Comment:
Printing a different string is certainly possible. Only requiring it for 2fa enabled accounts would require some non-trivial changes, essentially:

1. attempt the login without asking (nor sending) the OTP
2. if the login attempt fails, analyze the failure message and if the error is because an OTP is required, ask the user for it and retry step 1.